src/base/preferences.cpp

```diff
@@ -1265,12 +1265,20 @@ void Preferences::setMainGeometry(const QByteArray &geometry)
 
 QByteArray Preferences::getMainVSplitterState() const
 {
+#if (QT_VERSION >= QT_VERSION_CHECK(6, 0, 0))
+    return value("MainWindow/Qt6/VSplitterState").toByteArray();
```

Comment:
I really believe we shouldn't pollute main config file with all these GUI related states. But I know that even if you agree with me generally, you'll probably prefer to take the shortcut here. Then please let's at least do a little deeper refactoring to keep all the settings affected here in unified/consistent places. I suggest that they all have keys like `GUI/Qt6/<WidgetName>/<StateName>`.

Reply:
Done.